kubectl-plugin/pkg/cmd/session/session.go

```diff
@@ -3,51 +3,76 @@ package session
 import (
 	"context"
 	"fmt"
+	"strings"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util"
+	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util/client"
+	"github.com/spf13/cobra"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/cli-runtime/pkg/genericiooptions"
-	"k8s.io/client-go/kubernetes"
 	"k8s.io/kubectl/pkg/cmd/portforward"
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
-
-	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
 )
 
 const (
-	DASHBOARD_PORT = 8265
-	CLIENT_PORT    = 10001
+	dashboardPort = 8265
+	clientPort    = 10001
+	servePort     = 8000
 )
 
 type SessionOptions struct {
-	ioStreams    *genericiooptions.IOStreams
 	configFlags  *genericclioptions.ConfigFlags
+	ioStreams    *genericiooptions.IOStreams
+	client       client.Client
+	ResourceType util.ResourceType
 	ResourceName string
 	Namespace    string
 }
 
+var (
+	sessionLong = templates.LongDesc(`
+		Forward local ports to the Ray resources.
+
+		Forward different local ports depending on the resource type: RayCluster, RayJob, or RayService.
+	`)
+
+	sessionExample = templates.Examples(`
+		# Forward local ports to the RayCluster resource
+		kubectl ray session raycluster/my-raycluster
```

Comment:
Will this default to raycluster if the resource type is not specified?

Reply:
No. Now the resource type is required.

Reply:
> Do you think it is better to default to RayCluster if the resource type is not specified?

Personally, yes I think so. Similar to how `kubectl logs` defaults the resource type to Pods, I think we should default the resource type to RayCluster

Reply:
Make senses to me. Updated in https://github.com/ray-project/kuberay/pull/2379/commits/e83ee5b5afcd8ebb99fa0212f44b2d731a88224f